config/a8c-for-agencies-development.json

```diff
@@ -46,7 +46,8 @@
 		"a4a-migrations-page-v2": true,
 		"a8c-for-agencies-agency-tier": true,
 		"a4a-pressable-referrals": true,
-		"a4a-updated-add-new-site": true
+		"a4a-updated-add-new-site": true,
+		"a4a-plugins-management": true
```

Comment:
Do we need the feature flag here? I think we can only use the section flag. What do you think?